code/game/objects/items/theft_items.dm

```diff
@@ -0,0 +1,295 @@
+	addtimer(CALLBACK(src, .proc/seal), 10 SECONDS)
+	return TRUE
+
+/obj/item/nuke_core_container/proc/seal()
+	if(istype(core))
+		STOP_PROCESSING(SSobj, core)
+		icon_state = "core_container_sealed"
+		playsound(src, 'sound/items/deconstruct.ogg', 60, TRUE)
+		if(ismob(loc))
+			to_chat(loc, "<span class='warning'>[src] is permanently sealed, [core]'s radiation is contained.</span>")
+
+/obj/item/nuke_core_container/attackby(obj/item/nuke_core/plutonium/core, mob/user)
+	if(istype(core))
+		if(!user.drop_item())
+			to_chat(user, "<span class='warning'>The [core] is stuck to your hand!</span>")
+			return
+		else
+			load(core, user)
+	else
+		return ..()
+
+//snowflake screwdriver, works as a key to start nuke theft, traitor only
+/obj/item/screwdriver/nuke
+	name = "screwdriver"
+	desc = "A screwdriver with an ultra thin tip that's carefully designed to boost screwing speed."
+	icon = 'icons/obj/nuke_tools.dmi'
+	icon_state = "screwdriver_nuke"
+	item_state = "screwdriver_nuke"
+	toolspeed = 0.5
+	random_color = FALSE
+
+/obj/item/paper/guides/antag/nuke_instructions
+	info = "How to break into a Nanotrasen nuclear device and remove its plutonium core:<br>\
+	<ul>\
+	<li>Use a screwdriver with a very thin tip (provided) to unscrew the terminal's front panel.</li>\
+	<li>Dislodge and remove the front panel with a crowbar.</li>\
+	<li>Cut the inner metal plate with a welding tool.</li>\
+	<li>Pry off the inner plate with a crowbar to expose the radioactive core.</li>\
+	<li>Pull the core out of the nuclear device. </li>\
+	<li>Put the core in the provided container, which will take some time to seal. </li>\
+	<li>???</li>\
+	</ul>"
+
+// STEALING SUPERMATTER.
+
+/obj/item/paper/guides/antag/supermatter_sliver
+	info = "How to safely extract a supermatter sliver:<br>\
+	<ul>\
+	<li>Approach an active supermatter crystal with radiation shielded personal protective equipment. DO NOT MAKE PHYSICAL CONTACT.</li>\
+	<li>Use a supermatter scalpel (provided) to slice off a sliver of the crystal.</li>\
+	<li>Use supermatter extraction tongs (also provided) to safely pick up the sliver you sliced off.</li>\
+	<li>Physical contact of any object with the sliver will dust the object, as well as yourself.</li>\
+	<li>Use the tongs to place the sliver into the provided container, which will take some time to seal.</li>\
+	<li>Get the hell out before the crystal delaminates.</li>\
+	<li>???</li>\
+	</ul>"
+
+/obj/item/nuke_core/supermatter_sliver
+	name = "supermatter sliver"
+	desc = "A tiny, highly volatile sliver of a supermatter crystal. Do not handle without protection!"
+	icon_state = "supermatter_sliver"
+	pulseicon = "supermatter_sliver_pulse"
+
+
+/obj/item/nuke_core/supermatter_sliver/attack_tk(mob/user) // no TK dusting memes
+	return
+
+
+/obj/item/nuke_core/supermatter_sliver/can_be_pulled(user) // no drag memes
+	return FALSE
+
+/obj/item/nuke_core/supermatter_sliver/attackby(obj/item/W, mob/living/user, params)
```

Comment:
Would you mind changing this from `W` to `I`? It doesn't make any actual difference obviously but as far as I know `W` is an old standard which is being phased out.
![image](https://user-images.githubusercontent.com/57483089/126810292-e1a06cd3-ee7e-4948-ab6c-8f7a9efb4401.png)

Reply:
Sure